package.json

```diff
@@ -15,77 +15,82 @@
 		"start": "npm run build && docker-compose -p osctf -f docker/production.yml up",
 		"start-dev": "docker-compose -p osctf -f docker/development.yml up & npm run build-dev",
 		"server": "node --experimental-modules server/",
-		"server-dev": "nodemon --experimental-modules -e js,mjs,json,pug server/",
+		"server-dev": "nodemon -L --watch server/ --experimental-modules -e js,mjs,json,pug server/",
 		"build": "webpack --define process.env.NODE_ENV=\"production\" --config build/webpack.production.config.js",
 		"build-dev": "webpack-dev-server --config build/webpack.development.config.js",
 		"build-stats": "webpack --define process.env.NODE_ENV=\"production\" --config build/webpack.production.config.js --json > stats.json"
 	},
 	"dependencies": {
```

Comment:
Love the use of exact version numbers here.

Reply:
?

Also thanks for the feedback. :-)